staff/lluis-bagur/CV/index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css?family=PT+Sans" rel="stylesheet">
+    <title>CVlluisbaguur</title>
+    
+</head>
+<body>
+<main>
+    <header>
+      <h1>CV Lluis Bagur</h1>
+      <nav style="">
+    <ul>
+        <button><a href="#education" title="index.html">Historial academico</a></button>
+        <button><a href="#exp" title="index.html">Experiencia laboral</a></button>
+        <button><a href="#aficiones" title="index.html">Aficiones</a></button>
+    </ul>
+    </nav>
+    </header>
+
+    <section>
+        <h3>Datos personales:</h3>
+        <p>Nombre: lluis Bagur</p>
+        <p>Email: <a href="mailto:lluisbagur09@gmail.com">lluisbagur09@gmail.com</a></p>
+        <p>Teléfono: 6XX-XXX-XXX</p>
+    </section>
+       <img src="./profile.jpg" class="profileImage" style="width: 200px">
```

Comment:
The img tag has two required attributes: src and alt.
The class is not necessary, you must remove it